server/pbench/bin/gold/test-7.6.txt

```diff
@@ -1,60 +1,100 @@
-  '_type': 'pbench-run-toc-entry'},
- {'_id': '52d58c324725aec09cfabc732be41d5e',
-  '_index': 'pbench.2015-09',
-  '_op_type': 'create',
-  '_parent': 'c71b2c9c961ebd55f61273a6a020bdc6',
-  '_source': {'@timestamp': datetime.datetime(1900, 1, 1, 0, 0), 'directory': '/sysinfo/foo/'},
-  '_type': 'pbench-run-toc-entry'}]
+[
+    {
+        "_id": "c71b2c9c961ebd55f61273a6a020bdc6",
+        "_index": "pbench.results.2015-09",
+        "_op_type": "create",
+        "_source": {
+            "@metadata": {
+                "file-date": "2015-09-28",
+                "file-name": "/var/tmp/pbench-test-server/test-7.6.tar.xz",
+                "generated-by": "index-pbench",
+                "generated-by-version": "0.1.0.0",
+                "md5": "c71b2c9c961ebd55f61273a6a020bdc6",
+                "pbench-agent-version": "plugins:"
+            },
+            "@timestamp": "2015-09-21T19:31:12.673292621",
+            "host_tools_info": [
+                {
+                    "hostname": "alphaville",
+                    "hostname-f": null,
+                    "tools": {
+                        "iostat": "--interval=\"10\"",
+                        "mpstat": "--interval=\"10\"",
+                        "perf": "--record-opts=\"record -a --freq=100\"",
+                        "pidstat": "--interval=\"10\"",
+                        "proc-interrupts": "--interval=\"10\"",
+                        "proc-vmstat": "--interval=\"10\"",
+                        "sar": "--interval=\"10\"",
+                        "turbostat": "--interval=\"10\""
+                    }
+                }
+            ],
+            "run": {
+                "config": "",
+                "controller": "alphaville.usersys.redhat.com",
+                "date": "2015-09-21T19:31:08",
+                "end_run": "2015-09-21T19:32:12.673292621",
+                "id": "c71b2c9c961ebd55f61273a6a020bdc6",
+                "name": "testdir",
+                "script": "",
+                "start_run": "2015-09-21T19:31:12.673292621",
+                "tarball-dirname": "test-7.6",
+                "tarball-toc-prefix": "test-7.6"
+            },
+            "sosreports": [
+                {
+                    "hostname-f": "localhos",
+                    "hostname-s": "localhos",
+                    "md5": "ed66db45311691ba31f2eb99008589cb  sosreport.tar.xz",
+                    "name": "test-7.6/sysinfo/foo/end/sosreport.tar.xz"
+                }
+            ]
+        },
+        "_type": "pbench-run"
+    },
+    {
+        "_id": "c4da353f8c81cd78952e5e488a1b8dd5",
+        "_index": "pbench.results.2015-09",
+        "_op_type": "create",
+        "_parent": "c71b2c9c961ebd55f61273a6a020bdc6",
+        "_source": {
+            "@timestamp": "2015-09-21T19:31:12.673292621",
+            "directory": "/",
+            "files": [
+                {
+                    "mode": 436,
```

Comment:
I pushed a fix to the dsa wip-indexing branch (although I meant to push it to my branch - sorry about that). Let me know if you see any problem.